samples/boards/intel_s1000_crb/intel_s1000_crb.rst

```diff
@@ -0,0 +1,10 @@
+.. _intel_s1000_crb-samples:
+
+Intel S1000 Customer Reference Board Samples
```

Comment:
Since this is plural ("samples") will there be more coming, and is the "Customer Reference Board" available to the public?

Reply:
Yes. Board availability only under NDA.